modules/rostests/apitests/user32/PrivateExtractIcons.c

```diff
@@ -3,41 +3,128 @@
  * LICENSE:         LGPLv2.1+ - See COPYING.LIB in the top level directory
  * PURPOSE:         Test for PrivateExtractIcons
  * PROGRAMMER:      Hermes Belusca-Maito
+ *                  Doug Lyons (douglyons@douglyons.com)
```

Comment:
tiny nitpick; https://reactos.org/wiki/Coding_Style suggests to use <> around the email address. I would suggest to stick to that for consistency reasons. LGTM otherwise.

Reply:
Done. Thanks.